posts/copilot-workspace/index.qmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "Some thoughts after a trial run of GitHub’s Copilot Workspace"
+author:
+  - name: "Joshua W. Lambert"
+    orcid: "0000-0001-5218-3046"
+  - name: "James Mba Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Pratik Gupte"
+    orcid: "0000-0001-5294-7819"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-06-24"
+categories: [AI, GitHub, Copilot Workspace, R, R package, software development]
```

Comment:
```suggestion
categories: [AI, GitHub, Copilot Workspace, R, R package, software development, DOI]
```

In case you'd like a DOI for this (or any future) post - add the DOI category. This is super new and haven't shared the procedure yet, but figured it would be relevant to mention here.